dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataLogsConnectorTest.java

```diff
@@ -56,6 +57,140 @@ public class TeradataLogsConnectorTest extends AbstractConnectorExecutionTest {
 
   private final TeradataLogsConnector connector = new TeradataLogsConnector();
 
+  @Test
+  public void addTaskTo_additionalTablesOnlyIfUserDefine() throws Exception {
+    // Option 1
```

Comment:
I recommend to split this test into 2: for RES_USAGE_SCPU_TABLE and for RES_USAGE_SPMA_TABLE to make the tests focused.

Reply:
done